### PR TITLE
Enhancement: Move DynamoDB logic, Creating required IAM roles, Add pr…

### PR DESCRIPTION
**Pull request by billsedison**
…oper logging at enter/exit of Lambda functions, Add proper error handling wherever needed, Remove hard-coded service URL for Teams Lambda, Fix lint errors

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -104,4 +117,19 @@ plugins:
 custom:
   serverless-offline-sns:
     port: 4000
-    debug: false
\ No newline at end of file
+    debug: false
+  sns:
+    tcSlackEvents: tc-slack-events
+    tcSlackInteractive: tc-slack-interactive
+  region: ${opt:region, self:provider.region}
```
Comment (RishiRajSahu) [inline serverless.yml]: @billsedison how does it gets region value ? we haven't provided region value under `provider` ?

File: src/common/dbHelper.js [hunk 2]
```diff
@@ -19,11 +21,15 @@ async function initialize () {
   })
 
   const dynamodb = new AWS.DynamoDB()
-  // Create table if it does not exist already
-  const list = await dynamodb.listTables().promise()
-  if (!list.TableNames.includes(config.get('DYNAMODB.PROJECT_TABLE_NAME'))) {
-    await dynamodb.createTable(projectsSchema).promise()
-  }
+
+  const projectsTable = config.get('DYNAMODB.PROJECT_TABLE_NAME')
```
Comment (RishiRajSahu) [inline src/common/dbHelper.js]: @billsedison I think we agreed on getting this created via serverless ? Please recheck if we have that as part of our winner submissions.

File: config/default.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ module.exports = {
   },
   // Winston log level
   LOG_LEVEL: 'error',
+  DISABLE_LAMBDA_DEBUG_LOGGING: false,
```
Comment (RishiRajSahu) [inline config/default.js]: @billsedison Does making it true will only enable logs for lambda functions ?

Comment (billsedison) [reply config/default.js]: Yes

**Pull request merged**